package.json

```diff
@@ -75,7 +75,8 @@
     "dist": "npm test && npm run clean && NODE_ENV='production' ./node_modules/.bin/gulp dist",
     "serve": "NODE_ENV='development' ./node_modules/.bin/gulp serve",
     "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp livereload",
-    "test": "./node_modules/.bin/gulp copy:external-plugins && jest"
+    "test": "./node_modules/.bin/gulp copy:external-plugins && jest",
+    "shrinkwrap": "npm shrinkwrap --dev && ./node_modules/.bin/gulp fixShrinkwrap"
```

Comment:
Why do we pass in the dev flag here?

Reply:
Without the -dev flag it only shrinkwraps the deps, not the dev-deps. We pass the flag into dcos-ui package.json, so I thought I would do it here too.